ADOPTERS.md

```diff
@@ -24,4 +24,5 @@ This is a list of adopters of using PowerShell in production or in their product
 * [System Frontier](https://systemfrontier.com/solutions/powershell/) provides dynamically generated web GUIs and REST APIs for PowerShell and other scripting languages.  
   Enable non-admins like help desk and tier 1 support teams to execute secure web based tools on any platform `without admin rights`.  
   Configure flexible RBAC permissions from an intuitive interface, without a complex learning curve.  
-  Script output along with all actions are audited. Manage up to 5,000 nodes for free with the [Community Edition](https://systemfrontier.com/solutions/community-edition/).  
+  Script output along with all actions are audited. Manage up to 5,000 nodes for free with the [Community Edition](https://systemfrontier.com/solutions/community-edition/).
+* [Azure Resource Manager Deployment Scripts](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-script-template) Complete the "last mile" of your Azure Resource Manager (ARM) template deployments with a Deployment Script, which enables you to run arbitrary powershell script in the context of a deployment. Designed to let you complete tasks that are should be part of a deployment, but are not possible in an ARM template today — for example, creating a Key Vault certificate or querying an external API for a new CIDR block. 
```

Comment:
The locale should be removed in the URL as the docs website will automatically fetch the correct one for the user.  Also split the line which will be rendered correctly.

```suggestion
* [Azure Resource Manager Deployment Scripts](https://docs.microsoft.com/azure/azure-resource-manager/templates/deployment-script-template) Complete the "last mile" of your Azure Resource Manager (ARM) template deployments with a Deployment Script, which enables you to run arbitrary PowerShell script in the context of a deployment.
  Designed to let you complete tasks that are should be part of a deployment, but are not possible in an ARM template today — for example, creating a Key Vault certificate or querying an external API for a new CIDR block. 
```